platform/wab/src/wab/client/components/grouping/VirtualTree.tsx

```diff
@@ -13,11 +13,13 @@ export interface RenderElementProps<T> {
     isOpen: boolean;
     toggleExpand: () => void;
   };
+  isSelected?: boolean;
 }
 
-export interface VirtualTreeRef {
+export interface VirtualTreeRef<T = unknown> {
```

Comment:
Not ideal, but I didn't want to bother the other VirtualTree callers and couldn't think of a better solution, though making `getVisibleNodes` generic could be simpler.